src/Explorer/Panes/SettingsPane/index.tsx

```diff
@@ -0,0 +1,408 @@
+      )}`
+    );
+
+    if (shouldShowGraphAutoVizOption) {
+      NotificationConsoleUtils.logConsoleInfo(
+        `Graph result will be displayed as ${
+          LocalStorageUtility.getEntryBoolean(StorageKey.IsGraphAutoVizDisabled) ? "JSON" : "Graph"
+        }`
+      );
+    }
+
+    NotificationConsoleUtils.logConsoleInfo(
+      `Updated query setting to ${LocalStorageUtility.getEntryString(StorageKey.SetPartitionKeyUndefined)}`
+    );
+
+    close();
+  };
+
+  const isCustomPageOptionSelected = () => {
+    return pageOption === Constants.Queries.CustomPageOption;
+  };
+
+  const isUnlimitedPageOptionSelected = () => {
+    return pageOption === Constants.Queries.UnlimitedPageOption;
+  };
+
+  const onUnlimitedPageOptionKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.UnlimitedPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onCustomPageOptionsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.CustomPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onJsonDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("true");
+      event.stopPropagation();
+    }
+  };
+
+  const onGraphDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("false");
+      event.stopPropagation();
+    }
+  };
+
+  const _loadSettings = () => {
+    setIsExecuting(true);
+    try {
+      setPageOption(
+        LocalStorageUtility.getEntryNumber(StorageKey.ActualItemPerPage) === Constants.Queries.unlimitedItemsPerPage
+          ? Constants.Queries.UnlimitedPageOption
+          : Constants.Queries.CustomPageOption
+      );
+      setCustomItemPerPage(LocalStorageUtility.getEntryNumber(StorageKey.CustomItemPerPage));
+    } catch (exception) {
+      setFormErrors("Unable to load your settings");
+      setFormErrorsDetails(exception);
+    } finally {
+      setIsExecuting(false);
+    }
+  };
+
+  return (
+    <div>
+      <div className="contextual-pane-out" />
+      <div className="contextual-pane" id="settingspane">
+        <div className="contextual-pane-in">
+          <form className="paneContentContainer" onSubmit={submit}>
+            <div className="firstdivbg headerline">
+              <span role="heading" aria-level={2}>
+                Settings
+              </span>
+              <div className="closeImg" role="button" aria-label="Close pane" tabIndex={0} onClick={closePanel}>
+                <img src="../../../images/close-black.svg" title="Close" alt="Close" />
```

Comment:
import the image directly at top and use here: `import closeBlack from "../../../images/close-black.svg"`

Reply:
done